### PR TITLE
Refine single blog post styling

### PR DESCRIPTION
**Pull request by broskoski**
Re #808 

screenshot 2017-07-28 12 53 39

### CODE REVIEW

File: apps/blog/stylesheets/show.styl [hunk 1]
```diff
@@ -41,16 +41,24 @@
       img
       iframe
         display block
-        max-width 100%
+        max-width 500px
```
Comment (dzucconi) [inline apps/blog/stylesheets/show.styl]: :|

Comment (dzucconi) [inline apps/blog/stylesheets/show.styl]: percentage please

Comment (broskoski) [inline apps/blog/stylesheets/show.styl]: Following directions, but can translate to ems

Comment (dzucconi) [inline apps/blog/stylesheets/show.styl]: Translate to percentage

**Pull request merged**